src/main/webapp/app/course/tutorial-groups/tutorial-groups-management/tutorial-groups/tutorial-groups-management/tutorial-groups-import-dialog/tutorial-groups-registration-import-dialog.component.html

```diff
@@ -1,42 +1,203 @@
+            </div>
+            <div class="table-wrapper-scroll-y scrollbar">
+                <table class="table table-striped table-bordered table-sm table-responsive mt-2">
+                    <thead>
+                        <tr>
+                            <th jhiTranslate="artemisApp.tutorialGroupImportDialog.table.columns.title"></th>
+                            <th jhiTranslate="artemisApp.tutorialGroupImportDialog.table.columns.registrationNumber"></th>
+                            <th jhiTranslate="artemisApp.tutorialGroupImportDialog.table.columns.firstName"></th>
+                            <th jhiTranslate="artemisApp.tutorialGroupImportDialog.table.columns.lastName"></th>
+                            <th jhiTranslate="artemisApp.tutorialGroupImportDialog.table.columns.campus"></th>
+                            <th jhiTranslate="artemisApp.tutorialGroupImportDialog.table.columns.language"></th>
+                            <th jhiTranslate="artemisApp.tutorialGroupImportDialog.table.columns.additionalInformation"></th>
+                            <th jhiTranslate="artemisApp.tutorialGroupImportDialog.table.columns.capacity"></th>
+                            <th jhiTranslate="artemisApp.tutorialGroupImportDialog.table.columns.isOnline"></th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <th jhiTranslate="artemisApp.tutorialGroupImportDialog.table.groupA"></th>
+                            <td>123456</td>
+                            <td>John</td>
+                            <td>Doe</td>
+                            <td>Main Campus</td>
+                            <td>English</td>
+                            <td>Extra info</td>
+                            <td>20</td>
+                            <td>true</td>
+                        </tr>
+                        <tr>
+                            <th jhiTranslate="artemisApp.tutorialGroupImportDialog.table.groupA"></th>
+                            <td>555555</td>
+                            <td>Jonas</td>
+                            <td>Krapfen</td>
+                            <td>Main Campus</td>
+                            <td>English</td>
+                            <td>Extra info</td>
+                            <td>20</td>
+                            <td>true</td>
+                        </tr>
+                        <tr>
+                            <th jhiTranslate="artemisApp.tutorialGroupImportDialog.table.groupB"></th>
+                            <td>789012</td>
+                            <td>Anna</td>
+                            <td>Smith</td>
+                            <td>City Campus</td>
+                            <td>German</td>
+                            <td>Extra info</td>
+                            <td>15</td>
+                            <td>false</td>
+                        </tr>
```

Comment:
I think that data driven approaches are usually better (less error prone due to types and no repetition in the HTML), however, for this simple static information I do not have a strong opinion - does not need to be changed

Reply:
not necessary